config/output_schemas/short_form_schema.toml

```diff
@@ -86,7 +86,7 @@ Deduced_Data_Type = "float64"
 old_name = "headcount_defence"
 Deduced_Data_Type = "float64"
 
-[q601]
+[post_code]
 old_name = "601"
 Deduced_Data_Type = "str"
 
```

Comment:
I'm interested to know where these changes came from, were they on DAP?

Reply:
Yes, this was updated to reflect DAP